.github/workflows/bootstrap.yml

```diff
@@ -45,6 +45,7 @@ jobs:
 
     - name: Run Tests
       run: |
+           ulimit -v 6291456; \
```

Comment:
Have there been any issues with the runner running out of memory? I can't remember

Reply:
this seems sus to me as well

Reply:
This hunk has been removed